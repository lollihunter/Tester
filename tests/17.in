20 18 20
1 1
4 16
2 8
1 5
6 13
8 18
15 9
6 16
15 12
11 1
4 13
11 3
17 15
4 5
7 13
2 2
15 10
19 8
14 2
20 18
