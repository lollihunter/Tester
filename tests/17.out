10
1 16 14 3 5 8 6 9 13 20 
